zproject/prod_settings_template.py

```diff
@@ -152,6 +152,21 @@
     # "zproject.backends.GenericOpenIdConnectBackend",  # Generic OIDC integration, setup below
 )
 
+## JWT-based API keys fetching.
+##
+## If you want to fetch API keys via JWT in Zulip, you need to set the
+## secret key and algorithm to use to validate JWT tokens received.
+JWT_FETCH_API_KEYS: Dict[str, Any] = {
```

Comment:
Hmm, I'm wondering if we really need a separate `JWT_FETCH_API_KEYS` dict when possibly the already existing  `JWT_AUTH_KEYS` could be used for this as well?  One is for authentication if configured, one is for fetching the API key, seems like kind of the same scope.

Reply:
Yeah I guess the decision to re-use that dict depends on the format that we want to adopt (hosts vs subdomain ids) for configuring the backend.
Check the ongoing discussion https://github.com/zulip/zulip/pull/22955#discussion_r971813590:

> should we decide to use a subdomain based conf, we suggest to revert to the already existing JWT_AUTH_KEYS dictionary, since the settings would be somehow duplicated

Reply:
Yeah let's just use `JWT_AUTH_KEYS` and potentially extend the format for that. That's the better name, and AFAIK there's no reason not to just make that setting work how we'd like it to.

Reply:
@timabbott so should I update the PR (code and tests) ?
IIUC we're agreeing on re-using  `JWT_AUTH_KEYS` and its current subdomain-based format.

Reply:
This has been added to the PR, I'm closing this conversation.